SampleApp/SampleApp.Android/SampleAutopilot.cs

```diff
@@ -28,7 +28,7 @@ public override void OnAirshipReady(UAirship airship)
             }
 
             UrbanAirshipListener airshipListener = new UrbanAirshipListener();
-            airship.PushManager.SetNotificationListener(airshipListener);
+            airship.PushManager.NotificationListener = airshipListener;
```

Comment:
this was a compiler error

Reply:
Why is the process of adding this listener different from the process of adding the other three? Shouldn't it be the same?

Reply:
Setters and getters are automatically bound to properties.